.circleci/config.yml

```diff
@@ -16,8 +20,21 @@ jobs:
           paths:
             - ~/.cache
 
+  terratest:
+    docker:
+      - image: *circleci_docker_primary
+        environment:
+          AWS_DEFAULT_REGION: us-west-2
```

Comment:
Is this needed?

Reply:
No, it's been removed.

Reply:
The random region assignment is also working without TERRATEST_REGION. Should we leave TERRATEST_REGION out?

Reply:
I'm okay with leaving it out for now.